docs/api/1.x/_details-patch-object.rst

```diff
@@ -60,3 +53,38 @@ See :ref:`api-permissions-payload`.
 .. ----------------
 
 .. If a read-only field is modified, a |status-400| error is returned.
+
+JSON Patch Operations
+---------------------
+
+`JSON-Patch <https://tools.ietf.org/html/rfc6902>`_ is a way to define a sequence
+of operations to be applied on a JSON object.
+
+It's possible to use ``JSON-Patch`` notation setting ``Content-Type: application/jso-patch+json``.
+
+When using this `patch` method, the request should be defined as a list of operations, for example:
+
+.. code-block:: javascript
+
+    [
+        { "op": "test", "path": "data/a", "value": "foo" },
+        { "op": "remove", "path": "/data/a" },
+        { "op": "add", "path": "/data/b", "value": [ "foo", "bar" ] },
+        { "op": "replace", "path": "/data/b", "value": 42 },
+        { "op": "move", "from": "/data/a", "path": "/data/c" },
+        { "op": "copy", "from": "/data/b", "path": "/data/d" }
+    ]
+
+For more information about each operation, please refer the 
+`JSON-Patch Specification <https://tools.ietf.org/html/rfc6902>`_.
+
+Permissions can also be altered using ``JSON-Patch``. Value is not used on permission operations
+and can be ommited.
```

Comment:
This is very useful when altering permissions since there is no need to get the current value before adding some principal to the list.